docs/reference/query-dsl/intervals-query.asciidoc

```diff
@@ -42,78 +50,110 @@ POST _search
+--
 
 [[intervals-match]]
-==== `match`
+==== `match` rule parameters
 
-The `match` rule matches analyzed text, and takes the following parameters:
+The `match` rule matches analyzed text.
 
-[horizontal]
 `query`::
-The text to match.
+(Required, string) Text you wish to find in the provided `<field>`.
+
 `max_gaps`::
-Specify a maximum number of gaps between the terms in the text.  Terms that
-appear further apart than this will not match. If unspecified, or set to -1,
-then there is no width restriction on the match.  If set to 0 then the terms
-must appear next to each other.
++
+--
+(Optional, integer) Maximum number of positions between the matching terms.
+Terms further apart than this are not considered matches. Defaults to
+`-1`.
+
+If unspecified or set to `-1`, there is no width restriction on the match. If
+set to `0`, the terms must appear next to each other.
+--
+
 `ordered`::
-Whether or not the terms must appear in their specified order.  Defaults to
-`false`
+(Optional, boolean) 
+If `true`, matching terms must appear in their specified order. Defaults to
+`false`.
+
 `analyzer`::
-Which analyzer should be used to analyze terms in the `query`.  By
-default, the search analyzer of the top-level field will be used.
+(Optional, string) <<analysis, analyzer>> used to analyze terms in the `query`.
+Defaults to the top-level `<field>`'s analyzer.
+
 `filter`::
-An optional <<interval_filter,interval filter>>
+(Optional, <<interval_filter,interval filter>> rule object) An optional interval
+filter.
+
 `use_field`::
-If specified, then match intervals from this field rather than the top-level field.
-Terms will be analyzed using the search analyzer from this field.  This allows you
-to search across multiple fields as if they were all the same field; for example,
-you could index the same text into stemmed and unstemmed fields, and search for
-stemmed tokens near unstemmed ones.
+(Optional, string) If specified, then match intervals from this
+field rather than the top-level `<field>`. Terms are analyzed using the
+search analyzer from this field. This allows you to search across multiple
+fields as if they were all the same field; for example, you could index the same
+text into stemmed and unstemmed fields, and search for stemmed tokens near
+unstemmed ones.
 
 [[intervals-prefix]]
-==== `prefix`
+==== `prefix` rule parameters
 
-The `prefix` rule finds terms that start with a specified prefix.  The prefix will
-expand to match at most 128 terms; if there are more matching terms in the index,
-then an error will be returned.  To avoid this limit, enable the
-<<index-prefixes,`index-prefixes`>> option on the field being searched.
+The `prefix` rule matches terms that start with a specified set of characters.
+This prefix can expand to match at most 128 terms. If the prefix matches more than
+128 terms, {es} returns an error. You can use the
+<<index-prefixes,`index-prefixes`>> option to avoid this limit.
```

Comment:
s/option to avoid/option on the field mapping to avoid/

Reply:
Fixed with 553a686a20c.